### PR TITLE
fix router loops

### PR DESCRIPTION
**Pull request by proddy**
not perfect, but it fixes the problem

### CODE REVIEW

File: interface/package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "react-dom": "latest",
     "react-dropzone": "^14.2.3",
     "react-icons": "^4.12.0",
-    "react-router-dom": "6.18.0",
+    "react-router-dom": "6.20.0",
```
Comment (MichaelDvP) [inline interface/package.json:46]: you can use ^ again=> `"^6.20.0"`

**Review by MichaelDvP — APPROVED** [review]
works good.

**Comment by proddy** [comment]
dom-router reverted there changes in 6.20.1 so we'll do the same

**Comment by MichaelDvP** [comment]
The absolute path works with 6.20.0 and 6.20.1. Is there any advantage in using relative pathes?

**Comment by proddy** [comment]
> The absolute path works with 6.20.0 and 6.20.1. Is there any advantage in using relative pathes?

not really, it just saves a few bytes of flash and makes the code a little easier to read. I was hoping the route matching would work again and catch invalid paths, so preventing the browser warnings when going to for example /dashboard/devices222. but it didn't. I can easily go back to the last implementation, its just a few seconds of work

**Pull request merged**